artichoke-backend/src/extn/core/string/string.rb

```diff
@@ -687,12 +687,11 @@ def to_str
   end
 
   def tr(from_str, to_str)
-    # TODO: Support character ranges c1-c2
```

Comment:
It seems like character ranges have been implemented already, therefore i removed this TODO comment

Reply:
The below use of `Regexp` is a hack. This does method does not properly validate that ranges are ascending.

For example, this must error:

```
[2.6.6] > "hello".tr("a-y", "z-b")
Traceback (most recent call last):
        5: from /usr/local/var/rbenv/versions/2.6.6/bin/irb:23:in `<main>'
        4: from /usr/local/var/rbenv/versions/2.6.6/bin/irb:23:in `load'
        3: from /usr/local/var/rbenv/versions/2.6.6/lib/ruby/gems/2.6.0/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        2: from (irb):6
        1: from (irb):6:in `tr'
ArgumentError (invalid range "z-b" in string transliteration)
```

Can you restore the comment?